cmd/podman/system/df.go

```diff
@@ -0,0 +1,284 @@
+package system
+
+import (
+	"fmt"
+	"html/template"
+	"io"
+	"os"
+	"strings"
+	"text/tabwriter"
+	"time"
+
+	"github.com/docker/go-units"
+
+	"github.com/containers/libpod/cmd/podman/validate"
+
+	"github.com/containers/libpod/cmd/podman/registry"
+	"github.com/containers/libpod/pkg/domain/entities"
+	"github.com/spf13/cobra"
+)
+
+var (
+	dfSystemDescription = `
+	podman system df
+
+	Show podman disk usage
+	`
+	dfSystemCommand = &cobra.Command{
+		Use:   "df",
+		Args:  validate.NoArgs,
+		Short: "Show podman disk usage",
+		Long:  dfSystemDescription,
+		RunE:  df,
+	}
+)
+
+var (
+	dfOptions entities.SystemDfOptions
+)
+
+func init() {
+	registry.Commands = append(registry.Commands, registry.CliCommand{
+		Mode:    []entities.EngineMode{entities.ABIMode},
+		Command: dfSystemCommand,
+		Parent:  systemCmd,
+	})
+	flags := dfSystemCommand.Flags()
+	flags.BoolVarP(&dfOptions.Verbose, "verbose", "v", false, "Show detailed information on space usage")
```

Comment:
I know this is pre-existing and also in the man page, but this probably ought to be "disk usage" to match the above.

Reply:
or maybe better yet, be like DF(1) and go with "disk space usages" through out.